### DISCUSSION TITLE
Internal Server Error: failed to process metrics correctly

### DISCUSSION THREAD

**Discussion by jayvdb** [lowlighter/metrics#581] [General]
https://metrics.lecoq.io/ "Metrics preview" and "Insights" for me (user: jayvdb) fail atm with
> An error occurred while generating metrics :(
> Internal Server Error: failed to process metrics correctly

**Comment by lowlighter** [comment]
Hello,
Your issue may eventually be solved once #584 lands 🙂 

Thanks for your patience 🙇
